cmd/update.go

```diff
@@ -90,6 +90,12 @@ var updateCmd = &cobra.Command{
 			// we want to use and save absolute path for component
 			dir, err := filepath.Abs(componentLocal)
 			checkError(err, "")
+			fileInfo, err := os.Stat(dir)
+			checkError(err, "")
+			if !fileInfo.IsDir() {
+				fmt.Println("Please provide path to a directory")
```

Comment:
It should be "Please provide a path to the directory"

Reply:
@cdrage Done